sdk/python/kfp/_client.py

```diff
@@ -39,7 +39,15 @@
 # initialized with host=<inverse proxy endpoint>.
 # Set to 55 mins to provide some safe margin.
 _GCP_ACCESS_TOKEN_TIMEOUT = datetime.timedelta(minutes=55)
-
+_FILTER_OPERATIONS = {"UNKNOWN": 0,
+    # Operators on scalar values. Only applies to one of |int_value|,
+    # |long_value|, |string_value| or |timestamp_value|.
+    "EQUALS" : 1,
+    "NOT_EQUALS" : 2,
```

Comment:
No, they are not I kept them on order to have a reference for the alternative since we don't have a proper helper file. Maybe we should remove it. Initially I only. set it to 1, but @Bobgy suggested that we added a mapping in order for readers of the code to understand what it meant.